### DISCUSSION TITLE
How i can use cyrillic in enumeration field?

### DISCUSSION THREAD

**Discussion by CtFelix** [strapi/strapi#7208] [Legacy Topics]
Hello
If i try to type somethind on cyrillic i see an error "The value does not match the regex." how can i fix it?

**Comment by lauriejim** [comment]
Hello! you will not be able to do that because of the GraphQL integration.
You will find all information there - https://github.com/strapi/strapi/discussions/4780

**Reply by soupette** [reply]
However if you are not using the GraphQL plugin and you are sure to never use it in your application you can still set the enum values directly in `./api/<modelName>/models/<ModelName>.settings.json` (it is not really recommend though)

**Reply by CtFelix** [reply]
Thank you!

**Reply by CtFelix** [reply]
> Hello! you will not be able to do that because of the GraphQL integration.
> You will find all information there - #4780

It may be possible to create some aliases for such tasks.

**Reply by lauriejim** [reply]
Thank you for the insight
